test/test_core.py

```diff
@@ -5908,29 +5881,21 @@ def test_sigaction_default(self, signal, exit_code, assert_identical):
       assert_returncode=exit_code
     )
 
-  @requires_node
   @crossplatform
-  @parameterized({
-    '': (['-DMEMFS'],),
-    'nodefs': (['-DNODEFS', '-lnodefs.js'],),
-    'noderawfs': (['-DNODERAWFS', '-sNODERAWFS'],)
-  })
-  def test_unistd_access(self, args):
-    self.emcc_args += args
-    nodefs = '-DNODEFS' in args or '-DNODERAWFS' in args
+  @with_all_fs
+  def test_unistd_access(self):
```

Comment:
The wasmfs version of this test seems to be failing.

Reply:
Ah, this was one where I snuck in an extra test config.  Revert to just nodefs_both.